pyfixest/utils/utils.py

```diff
@@ -354,42 +325,3 @@ def msqrt(C: np.ndarray) -> np.ndarray:
     return np.quantile(tmaxs, 1 - alpha)
 
 
-def _select_order_coefs(res: pd.DataFrame, keep: list, drop: list):
```

Comment:
this was added in my old simultaneous confint PR but eventually cloned to `dev_utils` so this was a duplicate afaict. Couldn't find anywhere this was used, but unit tests might say otherwise.

Reply:
Yes, it's a duplicate. Good catch!